modules/spacecraftinstruments/rendering/renderablemodelprojection.cpp

```diff
@@ -126,9 +129,56 @@ RenderableModelProjection::RenderableModelProjection(const ghoul::Dictionary& di
         dictionary,
         "RenderableModelProjection"
     );
-    using ghoul::Dictionary;
-    Dictionary geometryDictionary = dictionary.value<Dictionary>(keyGeometry);
-    _geometry = modelgeometry::ModelGeometry::createFromDictionary(geometryDictionary);
+
+    if (dictionary.hasKey(KeyGeomModelFile)) {
+        std::string file;
+
+        if (dictionary.hasValue<std::string>(KeyGeomModelFile)) {
+            // Handle single file
+            file = absPath(dictionary.value<std::string>(KeyGeomModelFile));
+            _geometry = ghoul::io::ModelReader::ref().loadModel(file, false, true);
+        }
+        else if (dictionary.hasValue<ghoul::Dictionary>(KeyGeomModelFile)) {
+            ghoul::Dictionary fileDictionary = dictionary.value<ghoul::Dictionary>(
+                KeyGeomModelFile
+            );
+            std::vector<std::unique_ptr<ghoul::modelgeometry::ModelGeometry>> geometries;
+
+            for (std::string_view k : fileDictionary.keys()) {
+                // Handle each file
+                file = absPath(fileDictionary.value<std::string>(k));
+                geometries.push_back(ghoul::io::ModelReader::ref().loadModel(
+                    file,
+                    false,
+                    true
+                ));
+            }
+
+            if (geometries.size() > 0) {
+                ghoul::modelgeometry::ModelGeometry combinedGeometry =
+                    std::move(*geometries[0].release());
```

Comment:
Same comment as above... I'm afraight that the unique_ptr might trigger it's `delete` before the `std::move()` can yeet the memory/variables